### ISSUE TITLE
MongoDB metrics not working anymore after datadog-agent update

### ISSUE THREAD

**Issue by ahbruinsma** [DataDog/dd-agent#3025]
After updating our servers, the mongodb metrics are not working anymore, i'm receiving the following error in the datadog logs:

`Nov 14 15:31:53 samantha dd.collector[2788]: ERROR (__init__.py:788): Check 'mongo' instance #0 failed#012Traceback (most recent call last):#012  File "/opt/datadog-agent/agent/checks/__init__.py", line 771, in run#012    self.check(copy.deepcopy(instance))#012  File "/opt/datadog-agent/agent/checks.d/mongo.py", line 734, in check#012    ops = db['$cmd.sys.inprog'].find_one()#012  File "/opt/datadog-agent/embedded/lib/python2.7/site-packages/pymongo/collection.py", line 1006, in find_one#012    for result in cursor.limit(-1):#012  File "/opt/datadog-agent/embedded/lib/python2.7/site-packages/pymongo/cursor.py", line 1097, in next#012    if len(self.__data) or self._refresh():#012  File "/opt/datadog-agent/embedded/lib/python2.7/site-packages/pymongo/cursor.py", line 1019, in _refresh#012    self.__read_concern))#012  File "/opt/datadog-agent/embedded/lib/python2.7/site-packages/pymongo/cursor.py", line 905, in __send_message#012    helpers._check_command_response(doc['data'][0])#012  File "/opt/datadog-agent/embedded/lib/python2.7/site-packages/pymongo/helpers.py", line 196, in _check_command_response#012    raise OperationFailure(msg % errmsg, code, response)#012OperationFailure: Invalid collection name: admin.$cmd.sys.inprog`

We're using mongo 3.2.9 and datadog-agent version 5.10.0

/etc/init.d/datadog-agent info gives us also :
```
mongo
    -----
      - instance #0 [ERROR]: 'Invalid collection name: admin.$cmd.sys.inprog'
      - Collected 0 metrics, 0 events & 1 service check
      - Dependencies:
          - pymongo: 3.2
```

Did something change in the security of the datadog user or a different collection or a different config in the YAML file?

Wirh kind regards,
Andries Bruinsma

**Comment by olivielpeau** [comment]
Hi @ahbruinsma and thanks for your feedback.

This is a known issue of the `5.10.0` mongo check with MongoDB>=3.2, and has been fixed in #3016. We'll release the fix with `5.10.1` (should be out this week), until then you can either downgrade your Agent or use the latest mongo check on master (https://github.com/DataDog/dd-agent/blob/master/checks.d/mongo.py).

I'll go ahead and close this issue, but let me know if you have any questions!

**Comment by stieg** [comment]
Hit this myself.  Can confirm that 5.10.1 fixes the issue for me.  Thank you for the quick turnaround.
